source/modulo_components/include/modulo_components/ComponentInterface.hpp

```diff
@@ -674,7 +685,8 @@ template<class NodeT>
 inline void ComponentInterface<NodeT>::add_tf_broadcaster() {
   if (this->tf_broadcaster_ == nullptr) {
     RCLCPP_DEBUG(this->get_logger(), "Adding TF broadcaster.");
-    this->tf_broadcaster_ = std::make_shared<tf2_ros::TransformBroadcaster>(this->shared_from_this());
+    console_bridge::setLogLevel(console_bridge::CONSOLE_BRIDGE_LOG_NONE);
```

Comment:
I take it this is used to suppress TF errors / warnings outside of the ros logger context?

Reply:
yes indeed, if we would fork rviz and do the same there, we wouldn't get any of these annoying logs anymore